docs/public/styles/index.css

```diff
@@ -437,6 +437,13 @@ h5:hover .anchor {
 	z-index: 200;
 	outline: none;
 	transition: background 0.3s;
+	animation: sidebar-slide 500ms ease forwards 1;
```

Comment:
I like the animation a lot, but it's showing up on the home page too. There we don't have a sidebar. Maybe we just need to trigger the animation when we detect a route change away from home?

Screenshot when I've just got to the home page. Haven't navigated anywhere yet.

<img width="1031" alt="Screenshot 2021-07-08 at 18 05 48" src="https://user-images.githubusercontent.com/1062408/124955161-21bf8000-e017-11eb-8a0b-ecb0273ca078.png">